<?xml version="1.0" encoding="UTF-8"?>
<testng-results retried="2" ignored="2" total="8" passed="2" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-02-14T23:12:01 IST" name="Suite1" finished-at="2025-02-14T23:12:34 IST" duration-ms="32990">
    <groups>
      <group name="ErrorValidations">
        <method signature="ErrorMessegeValidationTest.validateLoginPageErrorMessage(java.util.HashMap)[pri:0, instance:rahulshetty.tests.ErrorMessegeValidationTest@333291e3]" name="validateLoginPageErrorMessage" class="rahulshetty.tests.ErrorMessegeValidationTest"/>
      </group> <!-- ErrorValidations -->
      <group name="SubmitOrder">
        <method signature="StandAloneTest.submitOrder(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:rahulshetty.tests.StandAloneTest@747ddf94]" name="submitOrder" class="rahulshetty.tests.StandAloneTest"/>
      </group> <!-- SubmitOrder -->
    </groups>
    <test started-at="2025-02-14T23:12:01 IST" name="Test11" finished-at="2025-02-14T23:12:15 IST" duration-ms="13958">
      <class name="rahulshetty.tests.ErrorMessegeValidationTest">
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:rahulshetty.tests.ErrorMessegeValidationTest@333291e3]" started-at="2025-02-14T23:12:01 IST" name="launchApplication" finished-at="2025-02-14T23:12:10 IST" duration-ms="9696" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="validateLoginPageErrorMessage(java.util.HashMap)[pri:0, instance:rahulshetty.tests.ErrorMessegeValidationTest@333291e3]" started-at="2025-02-14T23:12:10 IST" name="validateLoginPageErrorMessage" data-provider="getHash" finished-at="2025-02-14T23:12:11 IST" duration-ms="759" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{pass=123456, email=arunnatikar@gmail.com}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateLoginPageErrorMessage -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:rahulshetty.tests.ErrorMessegeValidationTest@333291e3]" started-at="2025-02-14T23:12:11 IST" name="tearDown" finished-at="2025-02-14T23:12:11 IST" duration-ms="182" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:rahulshetty.tests.ErrorMessegeValidationTest@333291e3]" started-at="2025-02-14T23:12:11 IST" name="launchApplication" finished-at="2025-02-14T23:12:14 IST" duration-ms="2261" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="validateLoginPageErrorMessage(java.util.HashMap)[pri:0, instance:rahulshetty.tests.ErrorMessegeValidationTest@333291e3]" started-at="2025-02-14T23:12:14 IST" name="validateLoginPageErrorMessage" data-provider="getHash" finished-at="2025-02-14T23:12:14 IST" duration-ms="760" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{pass=87969, email=arunnatikar22@gmail.com}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateLoginPageErrorMessage -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:rahulshetty.tests.ErrorMessegeValidationTest@333291e3]" started-at="2025-02-14T23:12:14 IST" name="tearDown" finished-at="2025-02-14T23:12:15 IST" duration-ms="126" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- rahulshetty.tests.ErrorMessegeValidationTest -->
    </test> <!-- Test11 -->
    <test started-at="2025-02-14T23:12:15 IST" name="Test6" finished-at="2025-02-14T23:12:34 IST" duration-ms="19014">
      <class name="rahulshetty.tests.StandAloneTest">
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:rahulshetty.tests.StandAloneTest@747ddf94]" started-at="2025-02-14T23:12:15 IST" name="launchApplication" finished-at="2025-02-14T23:12:19 IST" duration-ms="4361" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method retried="true" signature="submitOrder(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:rahulshetty.tests.StandAloneTest@747ddf94]" started-at="2025-02-14T23:12:19 IST" name="submitOrder" data-provider="getData" finished-at="2025-02-14T23:12:20 IST" duration-ms="804" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[arunnatikar99@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Rahul@1234]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[IPHONE]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[Cannot invoke "org.openqa.selenium.WebElement.findElement(org.openqa.selenium.By)" because "prod1" is null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: Cannot invoke "org.openqa.selenium.WebElement.findElement(org.openqa.selenium.By)" because "prod1" is null
at rahulshetty.pageobject.ProductCataloguePage.addProductToCart(ProductCataloguePage.java:62)
at rahulshetty.tests.StandAloneTest.submitOrder(StandAloneTest.java:37)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- submitOrder -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:rahulshetty.tests.StandAloneTest@747ddf94]" started-at="2025-02-14T23:12:20 IST" name="tearDown" finished-at="2025-02-14T23:12:20 IST" duration-ms="115" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:rahulshetty.tests.StandAloneTest@747ddf94]" started-at="2025-02-14T23:12:20 IST" name="launchApplication" finished-at="2025-02-14T23:12:26 IST" duration-ms="6626" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="submitOrder(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:rahulshetty.tests.StandAloneTest@747ddf94]" started-at="2025-02-14T23:12:26 IST" name="submitOrder" data-provider="getData" finished-at="2025-02-14T23:12:27 IST" duration-ms="761" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[arunnatikar99@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Rahul@1234]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[IPHONE]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[Cannot invoke "org.openqa.selenium.WebElement.findElement(org.openqa.selenium.By)" because "prod1" is null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: Cannot invoke "org.openqa.selenium.WebElement.findElement(org.openqa.selenium.By)" because "prod1" is null
at rahulshetty.pageobject.ProductCataloguePage.addProductToCart(ProductCataloguePage.java:62)
at rahulshetty.tests.StandAloneTest.submitOrder(StandAloneTest.java:37)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:269)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- submitOrder -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:rahulshetty.tests.StandAloneTest@747ddf94]" started-at="2025-02-14T23:12:27 IST" name="tearDown" finished-at="2025-02-14T23:12:27 IST" duration-ms="182" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:rahulshetty.tests.StandAloneTest@747ddf94]" started-at="2025-02-14T23:12:27 IST" name="launchApplication" finished-at="2025-02-14T23:12:30 IST" duration-ms="2420" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method retried="true" signature="submitOrder(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:rahulshetty.tests.StandAloneTest@747ddf94]" started-at="2025-02-14T23:12:30 IST" name="submitOrder" data-provider="getData" finished-at="2025-02-14T23:12:31 IST" duration-ms="774" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[arunnatikar99@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Rahul@1234]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[ADIDAS]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[Cannot invoke "org.openqa.selenium.WebElement.findElement(org.openqa.selenium.By)" because "prod1" is null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: Cannot invoke "org.openqa.selenium.WebElement.findElement(org.openqa.selenium.By)" because "prod1" is null
at rahulshetty.pageobject.ProductCataloguePage.addProductToCart(ProductCataloguePage.java:62)
at rahulshetty.tests.StandAloneTest.submitOrder(StandAloneTest.java:37)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- submitOrder -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:rahulshetty.tests.StandAloneTest@747ddf94]" started-at="2025-02-14T23:12:31 IST" name="tearDown" finished-at="2025-02-14T23:12:31 IST" duration-ms="193" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:rahulshetty.tests.StandAloneTest@747ddf94]" started-at="2025-02-14T23:12:31 IST" name="launchApplication" finished-at="2025-02-14T23:12:33 IST" duration-ms="1834" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="submitOrder(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:rahulshetty.tests.StandAloneTest@747ddf94]" started-at="2025-02-14T23:12:33 IST" name="submitOrder" data-provider="getData" finished-at="2025-02-14T23:12:33 IST" duration-ms="803" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[arunnatikar99@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Rahul@1234]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[ADIDAS]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[Cannot invoke "org.openqa.selenium.WebElement.findElement(org.openqa.selenium.By)" because "prod1" is null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: Cannot invoke "org.openqa.selenium.WebElement.findElement(org.openqa.selenium.By)" because "prod1" is null
at rahulshetty.pageobject.ProductCataloguePage.addProductToCart(ProductCataloguePage.java:62)
at rahulshetty.tests.StandAloneTest.submitOrder(StandAloneTest.java:37)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:269)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- submitOrder -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:rahulshetty.tests.StandAloneTest@747ddf94]" started-at="2025-02-14T23:12:33 IST" name="tearDown" finished-at="2025-02-14T23:12:34 IST" duration-ms="135" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- rahulshetty.tests.StandAloneTest -->
    </test> <!-- Test6 -->
  </suite> <!-- Suite1 -->
</testng-results>
